MakeMsix.targets

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Define the property MakeMsixVerbosity=high for additional logging (default=normal) -->
+  <PropertyGroup>
+    <MakeMsixVerbosity Condition="'$(MakeMsixVerbosity)' == ''">normal</MakeMsixVerbosity>
```

Comment:
Is there any sane way to determine a more dynamic default than 'normal' based on the MSBuild verbosity?

Reply:
Sounds like no, intentionally

https://social.msdn.microsoft.com/Forums/en-US/510f07b4-c5f7-43a8-b7cb-e3c398841725/getting-the-verbosity-level-from-a-task?forum=msbuild

>Is there a way for a Task (or, better, using a property) to determine the current verbosity level?
>
>You can't -- we didn't want tasks/build process to assume what loggers would want. Otherwise, eg., I wouldn't be able to walk up to any build process and attach my "super duper log everything and filter it later SQL database logger".

and https://social.msdn.microsoft.com/Forums/vstudio/en-US/0d4d53f9-ced2-4af9-a4b4-4a9694523492/condition-for-output-verbosity-level

>Tasks can't see logger verbosity. Verbosity is purely something the engine passes on to loggers, they can choose what they do with it. The reasoning is that if tasks understood verbosity, I couldn't write a logger that logged everything on low verbosity, for example. Also in future verbosity might be per-logger.
>
>I suggest that you have some other toggle for your exec task, using a property (you could set the property from the cmd line with /p).

and https://stackoverflow.com/questions/4341155/get-the-current-logging-verbosity-level-in-my-msbuild-script

Reply:
None of this is a Task.  Tasks are things like <Exec> which are exposed functionality from a Managed Assembly.  That being said, I'm sure those points are all still valid as they seem to point to a purposeful design here.  

I think this change kind of goes against that design though philosophy though.  This property makes all of the below output set to a certain level.  There are no varying levels of verbosity here for the logger to choose from.  This is because all of the logging is tied to this property.  So maybe the property is redundant, and the levels of each message should be set accordingly.  This allows the logger to actually decide the verbosity.

I only Unresolved this conversation because I don't think it's a done conversation.  I would not consider it blocking for your change or my signoff.  I'm just trying to help make your logging better.  

You should consider having more than one level of verbosity from your Target.  If you want a property similar to this (still beyond the built in one that works via the loggers), then you could have even more output exposed by the Target at the Info logging level.

Reply:
I defined the property consistent with other like implementations e.g. Microsoft.Windows.CppWinRT.targets in the CppWinRT package

```
    <PropertyGroup>
        <CppWinRTVerbosity Condition="'$(CppWinRTVerbosity)' == ''">normal</CppWinRTVerbosity>
        <CppWinRTCommandVerbosity Condition="'$(CppWinRTVerbosity)' == 'high'">-verbose</CppWinRTCommandVerbosity>
...
        <Message Text="GetResolvedWinMD: @(WinMDFullPath->'%(FullPath)')" Importance="$(CppWinRTVerbosity)"/>
```

Why's C++/WinRT do it? @kennykerr @BenJKuhn ?

Reply:
@Scottj1s may know.

Reply:
Updated MakeMsix.targets with specific log levels and removed MakeMsixVerbosity